### DISCUSSION TITLE
[PSD1] Esercizi Slide n. 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#114] [Esercizi (A-L)]
1. Convertire i seguenti numeri in decimale

	(a) $1110_4$
	(b) $401_5$
	(c) $932_{12}$
	(d) $347_8$

2. Convertire i seguenti numeri in base 2

	(a) $3112_4$
	(b) $401_8$
	(c) $732_8$
	(d) $347_8$

**Comment by Elia-Belli** [comment]
Considero il formato Unsigned
- **In Decimale**
(a) $1110_4 = (4^3+4^2+4) = 84_{10}$
(b) $401_5 = (4* 5^2+1) = 101_{10}$
(c) $932_{12} = (9* 12^2+3* 12+2) = 1334_{10}$
(d) $347_8 = (3* 8^2+4* 8+7) = 231_{10}$
- **In Binario**
(a) $3112_4 = 11\\_01\\_01\\_10_2$
(b) $401_8 = 100\\_000\\_001_2$
(c) $732_8 = 111\\_011\\_010_2$
(d) $347_8 = 011\\_100\\_111_2$
